pkg/graph/resolvers/system_intake_grb_reviewer.go

```diff
@@ -249,3 +249,25 @@ func GetPrincipalAsGRBReviewerBySystemIntakeID(ctx context.Context, systemIntake
 	}
 	return nil, nil
 }
+
+// UpdateSystemIntakeGRBReviewType updates the GRB review type on a system intake
+func UpdateSystemIntakeGRBReviewType(
+	ctx context.Context,
+	store *storage.Store,
+	input models.UpdateSystemIntakeGRBReviewTypeInput,
+) (*models.UpdateSystemIntakePayload, error) {
+	return sqlutils.WithTransactionRet(ctx, store, func(tx *sqlx.Tx) (*models.UpdateSystemIntakePayload, error) {
```

Comment:
Hmm, is there a reason that this takes a transaction / happens in a transaction blocks? Since this is only one action taking place, it might make since to convert the `TX` param to a named preparer (so it supports `tx` or `store` when a transaction is not needed)

Reply:
The reasoning was: to maintain homogeneity with neighboring logic and, to provide a more convenient region of code should we need to add more actions to this logic later. Is there an extra cost to using transactions?

Reply:
Ultimately, it probably won't have a big performance effect. `SQLX` is probably doing something similar under the hood somewhere. 

I think it is mostly a code understanding cost (why is this in a transaction if it is only doing one thing?). Neighboring functions seem to have multiple points where the method can fail, so the changes might need to be rolled back. If it's not a known need for this feature, I'd default to the YAGNI principle and leave it out. At a minimum, I'd change the store method to take a named preparer so that it can pass a tx or a named preparer.